apps/office-addin/src/content/index.tsx

```diff
@@ -26,3 +23,6 @@ Office.initialize = function (reason) {
   isOfficeInitialized = true;
   render(App);
 };
+
+// HACK: enable for browser-based dev
```

Comment:
necessary when building the content page in the browser, as it is only rendered within Powerpoint otherwise